src/simple_openid_connect/data.py

```diff
@@ -440,6 +441,12 @@ class AuthenticationRequest(OpenidBaseModel):
     acr_values: Optional[List[str]] = None
     "OPTIONAL. Requested Authentication Context Class Reference values Space-separated string that specifies the acr values that the Authorization Server is being requested to use for processing this Authentication Request, with the values appearing in order of preference The Authentication Context Class satisfied by the authentication performed is returned as the acr Claim Value, as specified in Section 2 The acr Claim is requested as a Voluntary Claim by this parameter."
 
+    code_challenge: Optional[str] = None
+    "Optional. Code Challenge Code challenge used in the code challenge. This parameter is REQUIRED if code_challenge_method is present."
+
+    code_challenge_method: Optional[str] = None
+    "OPTIONAL. Code Challenge Method Code challenge method used in the code challenge. If the code_challenge is present, this parameter is REQUIRED."
```

Comment:
> Code Challenge Code challenge used in the code challenge

While I realize how one can come up with these word combinations, it is a bit too much spaghetti for my taste.
A rewording of the docstring would be appropriate.